### PR TITLE
Use the official Minecraft wiki as the source

### PR DESCRIPTION
**Pull request by jamierocks**
It is more reputable and official. I would have switched the images to use it to, but they generate them dynamically... perhaps someone should make an API for this :D

### CODE REVIEW

File: t/Minecraft.t [hunk 1]
```diff
@@ -98,8 +98,8 @@ sub make_structured_answer {
             image => 'https://duckduckgo.com/iu/?u=' . uri_escape( $recipe{'image'} )
         },
          meta => {
-            sourceName => "Minecraft XL",
-            sourceUrl => "http://www.minecraftxl.com/crafting-recipes/"
+            sourceName => "Minecraft Wiki",
+            sourceUrl => "http://minecraft.gamepedia.com/"
```
Comment (moollaza) [inline t/Minecraft.t]: Is there a specific page on crafting we can send them to? The link should provide more info about the specific result you're seeing.

**Comment by moollaza** [comment]
@jamierocks thanks for contributing! We really appreciate it.

File: t/Minecraft.t (hunk 1 above)
Comment (jamierocks) [inline t/Minecraft.t]: They are under http://minecraft.gamepedia.com/ eg http://minecraft.gamepedia.com/Compass

Comment (jamierocks) [inline t/Minecraft.t]: Although they do have: http://minecraft.gamepedia.com/Recipes

Comment (moollaza) [inline t/Minecraft.t]: @jamierocks alright, this seems like the best place to send them then? 

http://minecraft.gamepedia.com/Crafting#Complete_recipe_list

Comment (jamierocks) [inline t/Minecraft.t]: Yes, but only if you target crafting recipes. IIRC currently you only support them, so it should be fine.

Comment (moollaza) [inline t/Minecraft.t]: :+1: Awesome, please update and I'll be glad to merge.

**Comment by jamierocks** [comment]
Would you like me to merge the commits?

**Comment by jamierocks** [comment]
It seems I may also have to change more links eg https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/Minecraft.pm#L65 

I will clone later on and fix these :D

**Comment by jamierocks** [comment]
Done + merged commits :D

**Comment by mintsoft** [comment]
Thanks for the contribution @jamierocks

**Comment by jamierocks** [comment]
No problem, glad to be of help :D

**Comment by moollaza** [comment]
@jamierocks Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

@mintsoft thanks for merging :+1:

**Comment by jamierocks** [comment]
It's absolutely. I use DuckDuckGo all the time, and so want to improve it just as it does my online presence.

**Pull request merged**